client/packages/plugins/backendTypes/generated/TransformRequisitionLineOutput.ts

```diff
@@ -1,4 +1,5 @@
 // This file was generated by [ts-rs](https://github.com/Aleph-Alpha/ts-rs). Do not edit this file manually.
+import type { PluginDataRow } from "./PluginDataRow";
 import type { RequisitionLineRow } from "./RequisitionLineRow";
 
-export type TransformRequisitionLineOutput = { transformed_lines: Array<RequisitionLineRow>, };
+export type TransformRequisitionLineOutput = { transformed_lines: Array<RequisitionLineRow>, plugin_data: Array<PluginDataRow> | null, };
```

Comment:
Was hoping plugin_data would be optional rather then null, but this is ok for now